docs/api_requests/block-io-engine.md

```diff
@@ -22,6 +22,9 @@ with the `io_engine` field taking two possible values:
 The `Sync` variant is the default, in order to provide backwards compatibility
 with older Firecracker versions.
 
+**Note** [vhost-user block device](./block-vhost-user.md) is another option
```

Comment:
I don't quite believe that it is necessary, because having another process is already a lot and clear that it requires a separate discussion in the doc. I just left it here to have a link to make users aware.

Reply:
ok thanks